### ISSUE TITLE
Updating Vortice.Dxc to 1.9.117 breaks shader compilation

### ISSUE THREAD

**Issue by paulbartrum** [amerkoleci/Vortice.Windows#173]
Updating to Vortice.Dxc 1.9.117 breaks with the message:
```
warning: DXIL.dll not found.  Resulting DXIL will not be signed for use in release environments.
[CreateComputeShaderInvalidShaderBytecode] [Error] [StateCreation] : ID3D12Device::CreateComputeShader: Compute Shader is corrupt or in an unrecognized format.
```

Seems to be caused by this pull request by @manuelxmarquez: https://github.com/amerkoleci/Vortice.Windows/pull/156
Contrary to what the comment says, CopyNativeAssemblies_Vortice_Dxc was working on my setup (.NET 5) and restoring the missing targets file resolves the issue by copying dxil.dll into the same directory as the exe.

The suggested fix according to the PR is adding this to the project file:
```
        <RuntimeIdentifier>win-x64</RuntimeIdentifier>
        <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
```
This works, but results in a huge number of unrelated DLLs being copied to the output folder (~220 files).

If this is an intentional change, feel free to close.

**Comment by paulbartrum** [comment]
Hmm I might have spoken too soon. @manuelxmarquez your fix works by packing the native DLLs into /lib/netstandard2.0 in the nuget package, right? When I do that I get this warning when building:
```
warning MSB3246: Resolved file has a bad image, no metadata, or is otherwise inaccessible. System.BadImageFormatException: Assembly file 'C:\Users\pbartrum\.nuget\packages\vortice.dxc\1.9.117\lib\netstandard2.0\dxcompiler.dll' could not be opened -- PE image doesn't contain managed metadata.
```
Reading the docs this is because files in the lib folder are added as project references (as well as runtime dependencies). This obviously doesn't work for native DLLs, hence the warning.

**Comment by manuelxmarquez** [comment]
Yeah, I should have known that! I moved the assemblies to the build folder and brought back the targets file to copy them. I tested on a net5.0 project and on a multi targeted project to ensure it is copied properly to the output directory.

I did not put it in the runtimes folder because that would result in copying the assemblies to 2 locations for net5.0 when only one of them works. Hopefully this is the final fix.
